portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -187,38 +187,41 @@ const submissionPortalActivity = async (application) => {
  * @returns {Promise<boolean>} Facility(ies) was(were) updated or not
  */
 const checkCoverDateConfirmed = async (app, auditDetails) => {
-  let hasUpdated = false;
-
-  if (app) {
-    try {
-      const facilities = await getAllFacilitiesByDealId(app._id);
-      const notYetSubmittedToUKEF = !app.submissionCount;
-      const haveFacilities = facilities?.length > 0;
-      const isAIN = app.submissionType === CONSTANTS.DEAL.SUBMISSION_TYPE.AIN;
-
-      if (notYetSubmittedToUKEF && haveFacilities) {
-        // Iterate through issued facilities
-        facilities
-          .filter((f) => f.hasBeenIssued && !f.coverDateConfirmed)
-          .map(async (f) => {
-            hasUpdated = true;
-            await updateFacility(f._id, { coverDateConfirmed: Boolean(isAIN) }, auditDetails);
-          });
-
-        // Iterate through unissued facilities
-        facilities
-          .filter((f) => !f.hasBeenIssued && f.coverDateConfirmed)
-          .map(async (f) => {
-            hasUpdated = true;
-            await updateFacility(f._id, { coverDateConfirmed: false }, auditDetails);
-          });
-        return hasUpdated;
-      }
-    } catch (error) {
-      console.error('Unable to set coverDateConfirmed for AIN facilities. %o', error);
+  if (!app) {
+    return false;
+  }
+
+  try {
+    const facilities = await getAllFacilitiesByDealId(app._id);
+    const isSubmittedToUkef = !!app.submissionCount;
```

Comment:
Maybe could use use `>= 1` check instead of `!!`?

Reply:
I'm not modifying the actual functions here -- it was originally negated, and I double negated it. We're not wanting this PR to scope creep -- it could be that we care about `>=1` but I'm avoiding heavy changes

Reply:
Understood RE scope creep, but the `submisssionCount` check and variable name has already been changed? IMO `>= 1`  would be clearer

Reply:
updated